packages/layout/src/yoga/index.js

```diff
@@ -1,14 +1,13 @@
 /* eslint-disable import/prefer-default-export */
 
-import * as Yoga from 'yoga-layout';
+import Yoga from 'yoga-layout';
 
 export const loadYoga = async () => {
```

Comment:
Looked a little bit and it has some trickle-down changes needed, but this doesn't need to be `async` anymore, correct?

Reply:
Good eye. Thanks!